docs/src/basics/blockchain_types.md

```diff
@@ -43,3 +43,46 @@ let my_number: b256 = 0x00000000000000000000000000000000000000000000000000000000
 let my_contract_id: ContractId = ~ContractId::from(my_number);
 let forty_two: b256 = my_contract_id.into();
 ```
+
+## `Identity` Type
+
+The `Identity` type is an enum that allows for the handling of both `Address` and `ContractId` types. This is useful in cases where either type is needed.
+
+An `Identity` is implemented as follows.
+
+```sway
+pub enum Identity {
+    Address: Address,
+    ContractId: ContractId,
+}
+```
+
+Casting to an `Identity` must be done explicity:
+
+```sway
+let my_number: b256 = 0x000000000000000000000000000000000000000000000000000000000000002A;
+let my_address: Address = ~Address::from(my_number);
```

Comment:
I would simply this to just `~Address::from(0x000000000000000000000000000000000000000000000000000000000000002A)` for brevity.